ui/app/src/components/FormsEngine/controls/VideoPicker.tsx

```
@@ -0,0 +1,317 @@
+	};
+
+	return (
+		<>
+			<Menu
+				anchorEl={addMenuButtonRef.current}
+				open={addMenuOpen}
+				onClose={() => setAddMenuOpen(false)}
+				sx={{
+					[`.${menuItemClasses.root}`]: { pl: 3 }
+				}}
+				children={menuOptions}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)